sw/device/sca/ecc_serial.c

```diff
@@ -36,6 +36,25 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
+/**
+ * Number of bytes for ECDSA P-256 private keys, message digests, and point
+ * coordinates.
+ */
+static const size_t kEcc256NumBytes = 256 / 8;
+/**
+ * Number of 32b words for ECDSA P-256 private keys, message digests, and point
+ * coordinates.
+ */
+static const size_t kEcc256NumWords = kEcc256NumBytes / sizeof(uint32_t);
+
+// If the P-256 size is not an even multiple of `kOtbnWideWordNumWords`, the
+// caller would need to write extra zeroes to OTBN for most inputs to fill out
+// the last wide word and avoid errors when OTBN attempts to read an
+// uninitialized value from DMEM.
+static_assert(kEcc256NumWords % kOtbnWideWordNumWords == 0,
```

Comment:
Thanks a lot for updating them Jade! This line caused a compilation error: `sw/device/sca/ecc_serial.c:54:15: error: static_assert expression is not an integral constant expression`.

After commenting out that line, I was able to run the program on the FPGA and collect power traces. Except this line, everything seems to be fine.

Reply:
That makes sense. We can also ensure that thing from the Python side.